src/cli/commands/audit.js

```diff
@@ -19,6 +19,7 @@ const gzip = promisify(zlib.gzip);
 export type AuditOptions = {
   groups: Array<string>,
   level?: string,
+  muteIssues?: string[],
```

Comment:
could this be extended to be an object?

there's other fields that would be good to add,

`expiryDate`
`justification`

Not saying it's required for this PR, but if its made extensible from the beginning there wouldn't need to be a breaking change